apps/discovery_api/config/config.exs

```diff
@@ -37,7 +37,7 @@ config :discovery_api, DiscoveryApi.Quantum.Scheduler,
 
 config :mime, :types, %{
   "application/zip" => ["zip", "shp", "shapefile"],
-  "application/gtfs" => ["pb"]
+  "application/gtfs+protobuf" => ["gtfs"]
```

Comment:
Do we know how this will impact Discovery UI?

Reply:
Yeah, I'm interested to know. Probably not your fault if it breaks stuff b/c our mime type stuff is a dumpster show

Reply:
the issue is the TDG doesn't accept `application/gtfs` anymore so if we keep it how it is then the TDG returns `application/octet-stream` or whatever the default sourceFormat is. I did grep for any datasets in prod, staging and dev with the `application/gtfs` source format and it doesn't look like there are any.